src/lib/core/option/option.ts

```diff
@@ -99,17 +92,22 @@ export class MdOption extends _MdOptionMixinBase implements CanDisableRipple {
   get disabled() { return (this.group && this.group.disabled) || this._disabled; }
   set disabled(value: any) { this._disabled = coerceBooleanProperty(value); }
 
+  /** Whether ripples for the option are disabled. */
+  @Input()
```

Comment:
I don't think that this should be an Input. The only way to disable ripples on options should be programmatic. It also means that we won't have to coerce the value.